app/src/main/java/com/infomaniak/drive/ui/addFiles/ImportFilesDialog.kt

```diff
@@ -110,7 +113,19 @@ class ImportFilesDialog : DialogFragment() {
                 )
             }
         }
-        lifecycleScope.launch { lifecycle.withResumed { findNavController().popBackStack() } }
+        lifecycleScope.launch {
+            lifecycle.withResumed {
+                with(findNavController()) {
+                    popBackStack()
+                    findNavController().navigate(
+                        R.id.fileListFragment,
+                        FileListFragmentArgs(folderId = navArgs.folderId, folderName = navArgs.folderName).toBundle()
```

Comment:
Why do you navigate here ?

Reply:
If already in home, navigate, or do nothing.